File: DuDoong-Common/src/main/java/band/gosrock/common/consts/DuDoongStatic.java
package band.gosrock.common.consts;

public class DuDoongStatic {
    public static final String AUTH_HEADER = "Authorization";
    public static final String BEARER = "Bearer ";
    public static final String WITHDRAW_PREFIX = "DELETED:";
    public static final String TOKEN_ROLE = "role";
    public static final String TOKEN_TYPE = "type";
    public static final String TOKEN_ISSUER = "DuDoong";
    public static final String ACCESS_TOKEN = "ACCESS_TOKEN";
    public static final String REFRESH_TOKEN = "REFRESH_TOKEN";
    public static final String KR_YES = "예";
    public static final String KR_NO = "아니요";

    public static final int MILLI_TO_SECOND = 1000;
    public static final int BAD_REQUEST = 400;
    public static final int UNAUTHORIZED = 401;
    public static final int FORBIDDEN = 403;
    public static final int NOT_FOUND = 404;
    public static final int INTERNAL_SERVER = 500;

    public static final Long NO_START_NUMBER = 1000000L;
    public static final Long MINIMUM_PAYMENT_WON = 1000L;
    public static final Long ZERO = 0L;

    public static final String assetDomain = "https://asset.dudoong.com/";

    public static final String KAKAO_OAUTH_QUERY_STRING =
            "/oauth/authorize?client_id=%s&redirect_uri=%s&response_type=code";

    public static final String[] SwaggerPatterns = {
        "/swagger-resources/**", "/swagger-ui/**", "/v3/api-docs/**", "/v3/api-docs",
    };
}

File: DuDoong-Common/src/main/kotlin/band/gosrock/common/consts/DuDoongStatic.kt
package band.gosrock.common.consts

object DuDoongStatic {
    const val AUTH_HEADER = "Authorization"
    const val BEARER = "Bearer "
    const val WITHDRAW_PREFIX = "DELETED:"
    const val TOKEN_ROLE = "role"
    const val TOKEN_TYPE = "type"
    const val TOKEN_ISSUER = "DuDoong"
    const val ACCESS_TOKEN = "ACCESS_TOKEN"
    const val REFRESH_TOKEN = "REFRESH_TOKEN"
    const val KR_YES = "예"
    const val KR_NO = "아니요"

    const val MILLI_TO_SECOND = 1000
    const val BAD_REQUEST = 400
    const val UNAUTHORIZED = 401
    const val FORBIDDEN = 403
    const val NOT_FOUND = 404
    const val INTERNAL_SERVER = 500

    const val NO_START_NUMBER = 1000000L
    const val MINIMUM_PAYMENT_WON = 1000L
    const val ZERO = 0L

    const val assetDomain = "https://asset.dudoong.com/"

    const val KAKAO_OAUTH_QUERY_STRING =
        "/oauth/authorize?client_id=%s&redirect_uri=%s&response_type=code"

    @JvmField
    val SwaggerPatterns = arrayOf(
        "/swagger-resources/**",
        "/swagger-ui/**",
        "/v3/api-docs/**",
        "/v3/api-docs",
    )
}
